package.json

```diff
@@ -132,7 +132,7 @@
     "li": "npm install @ts-common/local-install --no-save && local-install"
   },
   "engines": {
-    "node": ">=12.0.0"
+    "node": ">=10.11.0"
```

Comment:
What versions of Node do you actually want to guarantee are compatible with `oav`?  Personally, I would use Node 14, since it's the oldest version of Node still supported.

Ideally, you should also run automated tests on all the versions of Node you claim to support.  So if `engines` specifies `node >= 14.0.0`, you should run tests on Node 14, 16, and 18.

Reply:
Let's keep node 12. Then later on we can retire 12 support with major version upgrade. we should run tests against different node version in CI pipeline.

Reply:
I personally think you can drop support for Node 12 (or any EOL Node version) in either a major or a minor/patch upgrade.

Per strict semver, dropping support for a Node version is probably considered a breaking change.

However, one could also argue:
* Packages should drop support for Node versions when they stop testing on those versions
* It's unreasonable to expect package maintainers to test on Node versions very long after they are EOL
* Packages should not be forced to upgrade their major versions when Node versions go EOL